src/table/utils/styling-utils.js

```diff
@@ -64,8 +65,38 @@ export function getHeadStyle(layout, theme) {
   return getBaseStyling(header, 'header', theme);
 }
 
+export function getContentHoverBackgroundColor(theme) {
+  const contentHoverBackgroundColor = theme.getStyle('object', 'straightTable.content.hover', 'backgroundColor');
+  const contentBackgroundColor = theme.getStyle('object', 'straightTable.content', 'backgroundColor');
+  const tableBackgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
+  const objectBackgroundColor = theme.getStyle('object', '', 'backgroundColor');
+  const isHoverBackgroundColorFromTheme = !(
+    contentHoverBackgroundColor === contentBackgroundColor ||
+    contentHoverBackgroundColor === tableBackgroundColor ||
+    contentHoverBackgroundColor === objectBackgroundColor
+  );
+
+  return isHoverBackgroundColorFromTheme ? contentHoverBackgroundColor : undefined;
+}
+
+export function getContentHoverFontColor(theme) {
+  const contentHoverFontColor = theme.getStyle('object', 'straightTable.content.hover', 'color');
+  const contentFontColor = theme.getStyle('object', 'straightTable.content', 'color');
+  const tableFontColor = theme.getStyle('object', 'straightTable', 'color');
+  const isHoverFontColorFromTheme = !(
+    contentHoverFontColor === contentFontColor || contentHoverFontColor === tableFontColor
+  );
+
+  return isHoverFontColorFromTheme ? contentHoverFontColor : undefined;
+}
+
 export function getBodyStyle(layout, theme) {
+  const contentHoverBackgroundColorFromTheme = getContentHoverBackgroundColor(theme);
```

Comment:
SInce we know that we everything in this function is relevant to the body/content, you can remove content from all variable names, makes it easier to read

Reply:
Sure